bundles/net.tourbook.cloud/src/net/tourbook/cloud/dropbox/PrefPageDropbox.java

```diff
@@ -144,9 +148,25 @@ private void createUI_20_TokensInformation(final Composite parent) {
 
       _group = new Group(parent, SWT.NONE);
       GridDataFactory.fillDefaults().grab(true, false).applyTo(_group);
-      _group.setText(Messages.Pref_CloudConnectivity_Dropbox_Tokens_Information_Group);
+      _group.setText(net.tourbook.cloud.Messages.Pref_CloudConnectivity_CloudAccount_Group);
       GridLayoutFactory.swtDefaults().numColumns(2).applyTo(_group);
       {
```

Comment:
I think that once I've already described this problem but I don't know where it was.

When using externalized strings from more than one `Messages.java` files, then the externalize string tool **do not work.**

![mt-externalize-strings](https://user-images.githubusercontent.com/1283445/104019398-c9b8d580-51bb-11eb-89eb-1ef520a69f46.png)

My solution is, which you can find in many places, extract strings from the 2nd and further Messages.java into constants and put them at the top of the file (so you can find them easier)

![mt-externalize-strings-from-other-locations](https://user-images.githubusercontent.com/1283445/104019408-cde4f300-51bb-11eb-8ef2-7262a7f2416d.png)


When you want to run the externalize string tool, first comment out these constants at the top (which causes an error), run the externalize string tool, after that revert the comments, someday I found this handy solution.

Reply:
Oh I forgot about that. I will address that in my next PR

Reply:
>When you want to run the externalize string tool, first comment out these constants at the top (which causes an error), run the externalize string tool, after that revert the comments, someday I found this handy solution.

It's a very "hacky" solution though, I am not a fan of those "hacks"

Reply:
The other solution would be to improve the string externalisation tool which is propably very difficult, you have to manage different Messages.java files.